src/main/rx/rx.h

```diff
@@ -82,12 +82,11 @@ extern int16_t rcData[MAX_SUPPORTED_RC_CHANNEL_COUNT];       // interval [1000;2
 #define RSSI_SCALE_DEFAULT 30
 
 typedef enum {
-    RX_FAILSAFE_MODE_AUTO = 0,
-    RX_FAILSAFE_MODE_HOLD,
-    RX_FAILSAFE_MODE_SET,
+    RX_FAILSAFE_MODE_HOLD = 0,
+    RX_FAILSAFE_MODE_AUTO_OR_SET,
 } rxFailsafeChannelMode_e;
 
-#define RX_FAILSAFE_MODE_COUNT 3
+#define RX_FAILSAFE_MODE_COUNT 4
```

Comment:
This doesn't make sense.  there are only 2 modes!  See the comment about multi dimensional array for mode letters.